tests/bluetooth/tester/src/l2cap.c

```diff
@@ -155,17 +164,20 @@ static void connect(u8_t *data, u16_t len)
 	}
 
 	chan->le.chan.ops = &l2cap_ops;
-	chan->le.rx.mtu = DATA_MTU;
+	chan->le.rx.mtu = sys_le16_to_cpu(cmd->mtu);
 
 	err = bt_l2cap_chan_connect(conn, &chan->le.chan, cmd->psm);
 	if (err < 0) {
 		goto fail;
 	}
 
-	rp.chan_id = chan->chan_id;
+	rp = (void *) buf;
```

Comment:
Minor: drop the space between the typecast

Reply:
Actually, I see this style is used also elsewhere in this c-file, so no need to change it here (it can be done as part of a later cleanup)

Reply:
done